portal/views/assessment_engine.py

```diff
@@ -1095,16 +1096,21 @@ def present_assessment(instrument_id):
         description: if missing valid OAuth token or bad `next` parameter
 
     """
+    AE_intervention = Intervention.query.filter_by(name='assessment_engine').one()
```

Comment:
I should probably provide better access to the static list of interventions by object - this query will certainly work for now.  Might replace the string 'assessment_engine' with INTERVENTION.ASSESSMENT_ENGINE

Replace the import of STATIC_INTERVENTIONS with INTERVENTION